cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-" f"{uuid.uuid4()}." f"{extension}"
```

Comment:
So hard.... Simplify this row
```suggestion
    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
```